queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/RequestCommands/ZeroReceiptCommand.cs

```diff
@@ -0,0 +1,17 @@
+﻿using fiskaltrust.Middleware.Contracts.Constants;
+using fiskaltrust.Middleware.Contracts.Repositories;
+using fiskaltrust.Middleware.Contracts.RequestCommands;
+using fiskaltrust.Middleware.Contracts.RequestCommands.Factories;
+using fiskaltrust.storage.V0;
+using Microsoft.Extensions.Logging;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.RequestCommands
+{
+    public class ZeroReceiptCommand : Contracts.RequestCommands.ZeroReceiptCommand
+    {
+        public ZeroReceiptCommand(ICountrySpecificSettings countryspecificSettings, IMiddlewareQueueItemRepository queueItemRepository, IRequestCommandFactory requestCommandFactory, ILogger<RequestCommand> logger, IActionJournalRepository actionJournalRepository) :
+            base(countryspecificSettings, queueItemRepository, requestCommandFactory, logger, actionJournalRepository)
+        {
+        }
+    }
```

Comment:
So not sure if the logic for recovering is in the baseclass but just leaving this here inc ase we forgot it :)

Reply:
It is in the base class 👌 we have these `ICountrySpecificSettings` which contains a field `ResendFailedReceipts` that determines if failed receipts are resent.

https://github.com/fiskaltrust/middleware/blob/2c285afbc67667ab87738c5683114af5f6478d89/queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Constants/CountrySpecificSettings.cs

As mentioned above receipts cannot fail in the default queue since we don't sign 😁